charts/thanos/templates/receive/service-headless.yaml

```
@@ -0,0 +1,27 @@
+{{- if .Values.receive.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-headless" (include "thanos.receiveFullname" .) }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "thanos.receiveLabels" . | nindent 4 }}
+    prometheus.io/service-monitor: "false"
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - name: grpc
+      port: 10907
+      targetPort: grpc
+    - name: http-remote-write
+      protocol: TCP
+      port: 10908
+      targetPort: remote-write
```

```suggestion
      targetPort: http-remote-write
```
I'd prefer the port names to match. Also is there a reason why haven't you used the same port schema as the rest of the chart?

---

I don't think this has been resolved.